targets/csharp/templates/API.cs.ejs

```diff
@@ -18,6 +18,7 @@ namespace PlayFab
 %>        public static async Task<PlayFabResult<<%- apiCall.result %>>> <%- apiCall.name %>Async(<%- apiCall.request %> request, object customData = null, Dictionary<string, string> extraHeaders = null)
         {
 <%- getRequestActions("            ", apiCall, false) %>
+<%- getCustomApiLogic("            ", apiCall) %>
```

Comment:
nitpick: I know this line follows current conventions, but it seems difficult to read exactly how many spaces we expect to tab in. 

Could we convert this parameter to an int? Or have a util function that can convert from an int to output that tabbing as a string? Would be a bit easier to read.

Reply:
I'd be more than glad to vote in favor of this! However, one thing I'm concerned about in regards to this PR is that since it's a bigger scale change to (all) make.js files, maybe we should address that separately later?